src/components/field-types/RadioFieldComponent.vue

```diff
@@ -40,7 +40,7 @@
     props: {
       value: {
         // ID of select field can be of type: Integer, Long, String etc.
-        type: [String, Number],
+        type: [String, Number, Boolean],
```

Comment:
why boolean in the FileField PR

Reply:
I ran into it when using a real EntityType for testing, I could create a separate bug fix PR, but felt like overkill